### PR TITLE
The Phantom Pain

### PR DESCRIPTION
**Pull request by Kurfursten**
:cl:
* bugfix: You can no longer remotely destroy an organ from someone by destroying an organ you removed from their body earlier (e.g.: that monkey brain you took out to put their brain in)

### CODE REVIEW

**Comment by SonixApache** [comment]
it was funny while it lasted

**Comment by Kurfursten** [comment]
You know, in things like this, the fix looks so small, but it was terrible to dig through and find the cause.

**Comment by Exxion** [comment]
Shouldn't `owner` be getting removed when the organ is removed? Or is it supposed to stay there?

File: code/modules/organs/organ_internal.dm [hunk 1]
```diff
@@ -19,7 +19,7 @@
 
 
 /datum/organ/internal/Destroy()
-	if(owner) //Just going to assume these lists are here because we have bigger problems than a runtime if they aren't
+	if(owner && owner.internal_organs.Find(src))
```
Comment (Exxion) [inline code/modules/organs/organ_internal.dm]:
```suggestion
	if(owner && (src in owner.internal_organs))
```
`in` is allegedly slightly faster, as `Find()` is a proc and also has more information to return

**Comment by jknpj** [comment]
Exxion has a point, why the owner ref?

**Comment by Kurfursten** [comment]
Owner points to the individual the organ came from, it's used in some things like determining if the organ was ever part of someone with a mind (for organ heist stuff)

**Comment by Exxion** [comment]
That's silly. How does that work with transplants?
Not that it matters for a bugfix PR

**Pull request merged**